server.go

```diff
@@ -15,21 +15,26 @@ import (
 )
 
 type OpenAPIConfig struct {
-	DisableSwagger   bool                              // If true, the server will not serve the Swagger UI nor the OpenAPI JSON spec
-	DisableSwaggerUI bool                              // If true, the server will not serve the Swagger UI
-	DisableLocalSave bool                              // If true, the server will not save the OpenAPI JSON spec locally
-	SwaggerUrl       string                            // URL to serve the swagger UI
-	UIHandler        func(specURL string) http.Handler // Handler to serve the OpenAPI UI from spec URL
-	JsonUrl          string                            // URL to serve the OpenAPI JSON spec
-	JsonFilePath     string                            // Local path to save the OpenAPI JSON spec
-	PrettyFormatJson bool                              // Pretty prints the OpenAPI spec with proper JSON indentation
+	EngineOpenAPIConfig
```

Comment:
@EwenQuim 

One issue I have with this structure is that it will place `OpenAPIConfig` options that will have no affect on Engines for other frameworks. I personally think we need a configuration object for our net/http server for such things (so everything that isn't in the current `EngineOpenAPIConfig`) should solely exist on `Server` structure or some other structure that we define that holds these values.